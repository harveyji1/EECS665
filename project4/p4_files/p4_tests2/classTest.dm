Person : class {
    name : int;
    age : int;
};

g : () int {
    a: Person;
}