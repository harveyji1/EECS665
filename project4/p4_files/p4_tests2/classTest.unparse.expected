Person{Person} : class {
    name{int} : int;
    age{int} : int;
};
g{()->int} : () int {
    a{Person} : Person{Person};
}
